xds/internal/balancer/priority/config.go

```diff
@@ -0,0 +1,59 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package priority
+
+import (
+	"encoding/json"
+	"fmt"
+
+	internalserviceconfig "google.golang.org/grpc/internal/serviceconfig"
+	"google.golang.org/grpc/serviceconfig"
+)
+
+type child struct {
+	Config *internalserviceconfig.BalancerConfig
+}
+
+type lbConfig struct {
+	serviceconfig.LoadBalancingConfig
+
+	Children   map[string]*child
```

Comment:
Also a comment here about the structure of the config would be useful so that one doesn't have to look at the design doc to understand that.